app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,33 @@
+/* universal */
+
+body {
+padding-top: 60px;
```

Comment:
wrong indent 
for ex:
![image](https://user-images.githubusercontent.com/56816494/130425407-a66e2200-e71c-45aa-97fa-04b6c4b74c13.png)